boltzrpc/boltzrpc.proto

```diff
@@ -418,17 +419,44 @@ enum IncludeSwaps {
     AUTO = 2;
 }
 
+message AnySwapInfo {
+    string id = 1;
+    SwapType type = 2;
+    Pair pair = 3;
+    SwapState state = 4;
+    optional string error = 5;
+    string status = 6;
+    // the "from" amount of the swap.
+    // its the expected amount sent to be to the lockup address for submarine and chain swaps and
+    // the invoice amount for reverse swaps.
+    // the "to" amount can be calculated by subtracting the service and network fee.
```

Comment:
Not sure if we should just make the life of the API consumer easier by doing that calculation for them. Would feel a lot more comfortable using that amount directly